Automattic/sensei
# Fix/sidebar icon

Fixes #4332 
The sidebar icon currently does not hover/focus the same way as the other icons in the dashboard do, and it also doesn't look the best using alternative colour schemes that WordPress provides.  This PR fixes that! 😄 

### Changes proposed in this Pull Request

* Switches the dashboard sidebar icon to use a font icon instead of an SVG

### Testing instructions

* Check out this PR
* Go to the Dashboard -> User -> Profile
* Check a few different colour schemes
* Visually see that the icon is changing accordingly (should match the other icons)
* Click on the Sensei sidebar page and make sure the `active` colours are also correct.

### Screenshot / Video

**Previously**:
![sensei-icon-previously](https://user-images.githubusercontent.com/3220162/143520265-976e37ea-fbe3-4d08-8662-27a2e9796357.gif)

**Now**
Switching colour schemes
![sensei-icon-fix](https://user-images.githubusercontent.com/3220162/143520104-053afd98-5e28-4855-a1ee-2d3f458f9f91.gif)

Active menu color:
<img width="232" alt="Screen Shot 2021-11-25 at 9 49 45 AM" src="https://user-images.githubusercontent.com/3220162/143520131-3f9c8b5a-c56e-4511-96fc-9072762046d3.png">

## Notes
Other things I explored:
- While it is possible to [use CSS filters](https://developer.mozilla.org/en-US/docs/Web/CSS/filter) to change the colour of an SVG, it get's complicated and I wasn't able to achieve all the exact colours needed.  This would also add work if the colour schemes ever changed.
- I also considered creating a new SVG for each colour scheme, but decided against it as using an icon font seems better supported and more future proof.
